README.md

```diff
@@ -0,0 +1,160 @@
+            settings_overrides=settings_overrides
+        )
+```
+
+`settings_overrides` accepts all arguments used by `django-leaflet` in its [LEAFLET_CONFIG](https://django-leaflet.readthedocs.io/en/latest/templates.html#configuration), in addition to the following:
+
+- `DEFAULT_ZOOM` and `DEFAULT_CENTER` are required.
+- `MAP_ID` sets the html `id` for your widget map. Useful if you have multiple maps on the page.
+- `MAP_HEIGHT` and `MAP_WIDTH` allow you to size the widget in your form. Must be used together.
+- This example uses the default values for `MAP_LAYER_STYLE` and `MAP_LAYER_STYLING`.
+
+`SelectedArea` is a model defined in `example/models.py`, and can be adjusted for your needs.
+
+## Importing Census data
+
+This repo includes an `example` application using geographical data for 2018 Census block groups in Chicago. Look at `data/Makefile` and `management/commands/import_data.py` to see the ETL pipeline for that data.
+
+If you'd like to use the same pipeline in your project but for a different area:
+
+1. Copy the `data` directory into your project
+2. Update `STATES` in `scripts/states.py`. If the area is in a new state, create a new entry in STATES for that state; otherwise, update the existing state.
+3. Update `all` in `data/Makefile` to match the form `$(DATA_DIR)/final/cb_2018_{state id}_bg_500k.shp` for each state you're importing shapefiles for.
+4. To delete and regenerate the shapefiles, run:
+
+```bash
+cd data && make clean && make
+```
+
+Or if your app is containerized:
+
+```bash
+docker-compose run --rm app make clean -f example/data/Makefile
+docker-compose run --rm app make -f example/data/Makefile
+```
+
+5. Run `python manage.py import_data` to import the new geographical data. Or if your app is containerized, run `docker-compose run --rm app python manage.py import_data`
+
+## Changing the data structure
+
+Though the example implementation of this widget uses Census block groups, we've intentionally left the door open to varied geographic data. By default the widget expects each area to have the attributes in `example.models.Area`:
+
+```python
+class Area(models.Model):
+    id = models.CharField(max_length=12, primary_key=True)
+    geom = geo_models.MultiPolygonField(srid=4269)
+```
+
+If that structure doesn't work for you, it can be changed through creating a new widget that inherits from `django_geomultiplechoice.widgets.GeoMultipleChoiceWidget` and modifying `get_features()`.
+
+## Local development
+
+This app requires [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) for local development.
+
+To install and get started, run the following commands in your shell:
+
+```
+# Clone the repo
+git clone git@github.com:datamade/django-geomultiplechoice.git
+
+# Move into the folder
+cd django-geomultiplechoice
+```
+
+Copy the local_settings example file
+
+```bash
+cp example/local_settings.example.py example/local_settings.py
+```
+
```

Comment:
Now that the example settings file is mounted into the app container by default, we can go ahead and remove these lines.

Reply:
Ah ty for the catch!